modules/iap/iap.tf

```diff
@@ -108,15 +108,10 @@ resource "helm_release" "iap" {
   }
 }
 
-resource "kubernetes_annotations" "annotation" {
-  api_version = "v1"
-  kind        = "Service"
-  metadata {
-    name      = var.k8s_backend_service_name
-    namespace = var.namespace
-  }
-  annotations = {
-    "beta.cloud.google.com/backend-config" = "{\"default\": \"${var.k8s_backend_config_name}\"}"
```

Comment:
This is needed! otherwise the redeploy will miss the annotation

Reply:
moved into application respective services

Reply:
Test change image and redeploy please.

Reply:
tested working fine with IAP on frontend and jupyter